src/pages/profile/Profile.tsx

```diff
@@ -0,0 +1,56 @@
+import { FC, useEffect, useState } from "react";
+import profilepic from "../../assets/profile.jpg";
+import { Button } from "@/common/button";
+import { Error } from "@/common/error";
+import { useParams } from "react-router-dom";
+
+export const Profile : FC = ()=>{
+    const [posts, setPosts] = useState([]);
+    const { id } = useParams()
+
+    useEffect(()=>{
+        console.log(id); //Get user by id
+    })
+
+    return(
+        <div>
+            <div className="flex items-center p-12">
+                <div className="px-12">
+                    <img className="w-40 h-40 rounded-[100%]" src={profilepic} alt="user profile" />
+                </div>
+                <div>
+                    <h1 className="font-bold text-3xl">Ny Hasina VAGNO</h1>
+                    <p className="text-sm text-light">@hasnyvagno</p>
+                    <p className="text-sm text-light">En savoir plus sur cette chaine</p>
+                    <div className="flex justify-between">
+                        <Button classNameAttr="text-xs bg-gray-800 hover:bg-gray-600 my-3 text-sm text-white py-2 px-4 rounded-[25px]"
+                        label="Personnaliser cette chaine"
+                        >
+                            {null}
+                        </Button>
+                        <Button classNameAttr="text-xs bg-gray-800 hover:bg-gray-600 m-3 text-sm text-white py-2 px-4 rounded-[25px]"
+                        label="Gerer les contenus"
+                        >
+                            {null}
+                        </Button>
+                    </div>
+                </div>
+            </div>
+            <hr />
+            {
+                posts.length == 0 && <div className="grid grid-cols-3 gap-4">
+                <div className="col-start-2 row-start-10 text-center text-xs">
+                    <Error others={<Button classNameAttr="text-xs bg-gray-800 hover:bg-gray-600 m-3 text-sm text-white py-2 px-4 rounded-[25px]"
```

Comment:
This isn't an error.
And we will use some other _shadcn-ui_ components for errors
- [Alert](https://ui.shadcn.com/docs/components/alert) for callout
- [Toast](https://ui.shadcn.com/docs/components/toast) for notification